### ISSUE TITLE
Missing status code details

### ISSUE THREAD

**Issue by Anthony-Gaudino** [cloud-annotations/docusaurus-plugin-openapi#58]
On the status codes, information like `description` is missing:

![missing_description](https://user-images.githubusercontent.com/2439728/143302632-b294753e-f299-4a77-9d31-dfe346f5ca99.png)

On README the status code description appears:
![](https://user-images.githubusercontent.com/4212769/85324376-b9e3d900-b497-11ea-9765-c42a8ad1ff61.png)

My `package.json`:
```json
{
  "name": "tdsdocs",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "docusaurus": "docusaurus",
    "start": "docusaurus start",
    "build": "docusaurus build",
    "swizzle": "docusaurus swizzle",
    "deploy": "docusaurus deploy",
    "clear": "docusaurus clear",
    "serve": "docusaurus serve",
    "write-translations": "docusaurus write-translations",
    "write-heading-ids": "docusaurus write-heading-ids",
    "watch": "nodemon -e yaml,js --exec \"openapi-comment-parser src docs/openapi.json && node src/index.js\"",
    "api": "openapi-comment-parser test openapi.json"
  },
  "dependencies": {
    "@docusaurus/core": "2.0.0-beta.9",
    "@docusaurus/preset-classic": "2.0.0-beta.9",
    "@mdx-js/react": "^1.6.21",
    "@svgr/webpack": "^5.5.0",
    "clsx": "^1.1.1",
    "docusaurus-plugin-openapi": "^0.1.0",
    "file-loader": "^6.2.0",
    "prism-react-renderer": "^1.2.1",
    "react": "^17.0.1",
    "react-dom": "^17.0.1",
    "url-loader": "^4.1.1"
  },
  "browserslist": {
    "production": [
      ">0.5%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "openapi-comment-parser": "^1.0.0"
  }
}

```

**Comment by bourdakos1** [comment]
What does your open api spec look like?

**Comment by Anthony-Gaudino** [comment]
> What does your open api spec look like?

Forgot to add it:
```json
{
  "openapi": "",
  "info": {
    "title": "",
    "version": ""
  },
  "paths": {
    "/users/{userId}": {
      "get": {
        "summary": "Returns a user by ID.",
        "parameters": [
          {
            "name": "userId",
            "in": "path",
            "description": "The ID of the user to return.",
            "required": true,
            "schema": {
              "type": "integer",
              "format": "int64"
            }
          }
        ],
        "responses": {
          "200": {
            "description": "A user object.",
            "content": {
              "application/json": {
                "schema": {
                  "$ref": "#/components/schemas/User"
                }
              }
            }
          },
          "400": {
            "description": "The specified user ID is invalid (not a number)."
          },
          "404": {
            "description": "A user with the specified ID was not found."
          },
          "default": {
            "description": "Unexpected error"
          }
        }
      }
    }
  },
  "components": {}
}
```

**Comment by bourdakos1** [comment]
It doesn't look like you're doing anything wrong on your end. I will see what I can do to try and fix this.

**Comment by bourdakos1** [comment]
This should be fixed in v0.1.1
